kotlinpoet/src/main/java/com/squareup/kotlinpoet/CodeBlock.kt

```diff
@@ -18,6 +18,8 @@
 package com.squareup.kotlinpoet
 
 import java.lang.reflect.Type
+import java.text.DecimalFormat
```

Comment:
Wondering if there are multiplatform-friendly Kotlin alternatives, while KotlinPoet currently won't work without a JVM, in future we'd potentially want it to be JDK-free.

Reply:
I think there currently is no real alternative, except string-foo... I'd guess that the `DecimalFormat` option is the best for the moment, since string-foo could cause more issues until the switch to multiplatform

Reply:
I don't think the string-foo would be a terrible approach. We can always switch to it when needed.

Reply:
Thats what I thought, yet I think, for the moment, it's just a safer bet to use the java given features. Do you think like this too?